web-wallet/src/routes/(welcome)/login/+page.svelte

```diff
@@ -3,40 +3,36 @@
 <script>
   import { mdiArrowLeft, mdiKeyOutline } from "@mdi/js";
   import { validateMnemonic } from "bip39";
-  import { setKey } from "lamb";
 
   import { Button, Card, Textbox } from "$lib/dusk/components";
   import { AppAnchorButton } from "$lib/components";
   import { goto } from "$lib/navigation";
-  import { settingsStore, walletStore } from "$lib/stores";
+  import {
+    mnemonicPhraseResetStore,
+    settingsStore,
+    walletStore,
+  } from "$lib/stores";
   import { decryptMnemonic, getSeedFromMnemonic } from "$lib/wallet";
   import loginInfoStorage from "$lib/services/loginInfoStorage";
   import { getWallet } from "$lib/services/wallet";
 
-  const notice = [
-    "Logging in to a new wallet will overwrite the current local wallet cache",
-    ", meaning that when you log in again with the previous mnemonic/",
-    "account you will need to wait for the wallet to sync.",
-  ].join("");
-
   /**
    * @typedef {import("@dusk-network/dusk-wallet-js").Wallet} Wallet
    */
 
+  const noExistingWalletDetectedErrorMessage = "No existing wallet detected";
+  const mismatchedWalletDetectedErrorMessage =
+    "Mismatched wallet address detected";
+
   /** @type {(wallet: Wallet) => Promise<Wallet>} */
   async function checkLocalData(wallet) {
     const defaultAddress = (await wallet.getPsks())[0];
     const currentAddress = $settingsStore.userId;
 
-    if (defaultAddress !== currentAddress) {
-      // eslint-disable-next-line no-alert
-      if (currentAddress && !window.confirm(notice)) {
-        throw new Error("Existing wallet detected");
-      }
-
-      await wallet.reset();
-      settingsStore.reset();
-      settingsStore.update(setKey("userId", defaultAddress));
+    if (!currentAddress) {
```

Comment:
Do we really need two different errors if the outcome is the same?

Reply:
I don't have a strong preference, we can simplify and have one error instead.